### DISCUSSION TITLE
Fetch is not defined

### DISCUSSION THREAD

**Discussion by richardryangarcia** [wevm/viem#147] [Question] [answered]
When following the basic example to get a block number, i get the error Fetch is not defined. Is there a min node version or some other node related detail that I might be missing? Thanks yall!
![image](https://user-images.githubusercontent.com/31261908/223444378-213e413c-4fc5-423e-b6db-43b23c5e92e7.png)

The repo Im using is here https://github.com/richardryangarcia/bookish-octo-palm-tree

**Comment by tmm** [comment, marked as answer]
Thanks for checking out viem!

If you are using less than Node 18, you need to polyfill `fetch`. Easy to do and more info here: https://viem.sh/docs/compatibility.html

**Reply by richardryangarcia** [reply]
Beautiful! thank you. upgrading the node version fixed the issue. Now Im rollin 😎

**Reply by jxom** [reply]
Rollin' rollin' rollin'.
